config/forcedPhotCcd.py

```diff
@@ -2,6 +2,7 @@
 
 config.measurement.load(os.path.join(os.path.dirname(__file__), "apertures.py"))
 config.measurement.load(os.path.join(os.path.dirname(__file__), "kron.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "gaap.py"))
```

Comment:
Don't turn on here:
1) no slow measurement plugin on any per-visit processing. 
2) Even if it were fast, I'm not sure it makes sense. 
Let's talk about it first.